hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/GrpcXceiverService.java

```diff
@@ -52,6 +107,7 @@ public StreamObserver<ContainerCommandRequestProto> send(
 
       @Override
       public void onNext(ContainerCommandRequestProto request) {
+        InputStream popStream = zeroCopyMessageMarshaller.popStream(request);
```

Comment:
Move `popStream` inside finally.  BTW, the `popStream(..)` method is going to be replaced by a new `release(..)` method; see https://github.com/apache/ratis/pull/971/files